### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by Eraden**

### CODE REVIEW

**Comment by Nereuxofficial** [comment]
Hey, thank you so much for implementing this! We really appreciate it. It looks good except for the holes being in a broken state and their indicator being present because our check for them is really primitive. I will review it in the following days :)

File: game/src/languages/german.rs [hunk 1]
```diff
@@ -68,23 +68,23 @@ pub const BUTTON_INFO: &str = "Bitte drücke die Leertaste!";
 pub const WINNING_TEXT: &str = "Du wurdest gerettet!";
 
 /// Constants for the events that can occur.
-pub const KOMETENEINSCHLAG: [&str; 2] = [
+pub const COMETA_STRIKE: [&str; 2] = [
```
Comment (Nereuxofficial) [inline game/src/languages/german.rs:71]: COMET_STRIKE*

**Review by Nereuxofficial — CHANGES_REQUESTED** [review]
The Pull Request looks great apart from a few minor issues and you can merge it once those are fixed:
- The English IntroScreen has Text that overruns to the right
- Some Internal Code sadly depends on the names of the Machines(due to time constraints in the project). So in event.rs Line 164 and in generate_machines.rs Line 34 we use kanguage-dependent variables so changing the language changes the behaviour.

Again thank you for taking the time to implement this

**Comment by Eraden** [comment]
Hi, I think I solved all issues here but unfortunately this ends up being huge PR

Here is what I did:

* Strings where replaces with enum for Machine and Trade, additionally this should prevent some problems with unmatched name and is a little bit faster
* Hash maps have capacity since overflowing then is really slow and requires rehashing every key
* Into screen was fixed
* Typos where removed

**Comment by Nereuxofficial** [comment]
> Note:
>
> Some inline attributes where removed. Be careful with them and note they are not c++ inline, they are only useful in libraries since they are instruction for compiler this part should be inline in user application not in current library

Ah so inlining is only useful for usage across crate boundaries. Thanks for pointing that out

**Review by Nereuxofficial — APPROVED** [review]
LGTM

**Comment by Eraden** [comment]
Hi, I can't merge or sign reviewer.

Additionally I created couple issues

**Pull request merged**